src/Client.php

```diff
@@ -164,8 +164,6 @@ public function sendRequest(RequestInterface $request)
         $response = $this->transport->sendRequest($request);
         $this->logger->info(sprintf("Response time in %.3f sec", microtime(true) - $start));       
 
-        $result = new Elasticsearch;
-        $result->setResponse($response, $request->getMethod() === 'HEAD' ? false : $this->getResponseException());
-        return $result;
+        return (new Elasticsearch())->setResponse($response, $request->getMethod() === 'HEAD' ? false : $this->getResponseException());     
```

Comment:
This code is very difficult to read, I'll prefer to leave as is. Simple is beautiful :-)